### PR TITLE
docs: update cloudflare worker deploy section

### PR DESCRIPTION
**Pull request by PaiJi**
Add section on deploying Cloudflare Worker via GitHub Action CI.

This is my first documentation contribution, please feel free to point out any errors or things that shouldn't be here, thanks!😊

### CODE REVIEW

File: getting-started/cloudflare-workers.md [hunk 1]
```diff
@@ -126,6 +126,44 @@ bun run deploy
 
 That's all!
 
+### Deploy from Github Action
```
Comment (yusukebe) [inline getting-started/cloudflare-workers.md:129]: Can you move this section to the bottom? I think the other sections are more important than this.

**Comment by yusukebe** [comment]
Hi @PaiJi 

Thanks! This is great! I've left the comment. Please check it.

File: getting-started/cloudflare-workers.md (hunk 1 above)
Comment (PaiJi) [reply getting-started/cloudflare-workers.md]: @yusukebe its done!

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@PaiJi 

Thanks! Shipping it.

**Pull request merged**